changelog.yaml

```diff
@@ -16,7 +16,23 @@
 
 - version: "NEXT"
   date: TBD
-  
+  changes:
+  - type: enhancement
+    impact: minor
+    title: Make Jenkinsfile Runner container image configurable in PipelineRun custom resource objects.
+
+    description: >-
+      Jenkins filerunner image is configurable in Steward-PipelineRun manifest at 'spec.image' now.

```

Comment:
#### Update:
Jenkins filerunner properties are configurable in Steward-PipelineRun manifest under 'spec.jenkinsfileRunner'

Reply:
Updated